proparse/src/main/java/org/prorefactor/treeparser/symbols/Widget.java

```diff
@@ -8,15 +8,17 @@
  * Contributors:
  *    John Green - initial API and implementation and/or initial documentation
  *******************************************************************************/ 
-package org.prorefactor.treeparser;
+package org.prorefactor.treeparser.symbols;
+
+import org.prorefactor.treeparser.TreeParserSymbolScope;
 
 /**
  * A Symbol defined with DEFINE &lt;widget-type&gt; or any of the other various syntaxes which implicitly define a widget.
  * This includes FRAMEs, WINDOWs, MENUs, etc.
  */
 public abstract class Widget extends Symbol implements IWidget {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this abstract class name to match the regular expression '^Abstract[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00118)